app/models/cdo/view/check-activities.js

```diff
@@ -1,7 +1,8 @@
-const { mapActivityDtoToCheckActivityRow } = require('../../mappers/check-activities')
+const { flatMapActivityDtoToCheckActivityRow } = require('../../mappers/check-activities')
 
 function ViewModel (cdo, activities, backNav) {
-  const mappedActivities = activities.map(mapActivityDtoToCheckActivityRow)
+  console.log('~~~~~~ Chris Debug ~~~~~~ ', 'Activities', JSON.stringify(activities))
```

Comment:
I presume you meant to remove the debug?

Reply:
Thanks, good catch

Reply:
fixed